ethereum/vyper
# More name change stuff

### - What I did
Finished grep foo'ing for `viper`

### - How I did it
I'll never tell!

### - How to verify it
More grep foo!

Also verify I didn't break `readthedocs` or test/build process somehow

### - Cute Animal Picture
![ninja cat](https://static.boredpanda.com/blog/wp-content/uploads/2017/10/ninja-cats-photography-hisakata-hiroyuki-59f19ad3a95f6__880.jpg)
